libraries/AP_RangeFinder/AP_RangeFinder_VL53L1X.h

```diff
@@ -1253,6 +1253,7 @@ class AP_RangeFinder_VL53L1X : public AP_RangeFinder_Backend
     // get a reading
     bool get_reading(uint16_t &reading_cm);
     AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev;
+    static const uint8_t MEASUREMENT_TIME_MS = 50; // Start continuous readings at a rate of one measurement every 50 ms
```

Comment:
should be in the single cpp file that uses it as a #define
header file is an interface to the outside world

Reply:
it shouldn't matter as here we are in the class scope